### DISCUSSION TITLE
zeppelin-interpreter 打包不包含依赖包

### DISCUSSION THREAD

**Discussion by kevin-xin** [apache/iotdb#3722] [Q&A] [answered]
打包方式使用的官方步骤https://iotdb.apache.org/zh/UserGuide/V0.12.x/Ecosystem%20Integration/Zeppelin-IoTDB.html
使用的源码发布的0.12.1
使用的命令：mvn clean package -pl zeppelin-interpreter -am -DskipTests
在zeppelin-interpreter 下只生成了 zeppelin-iotdb-0.12.1.jar 和 zeppelin-iotdb-0.12.1-sources.jar ，没有生成zeppelin-{version}-SNAPSHOT-jar-with-dependencies.jar

**Comment by HTHou** [comment, marked as answer]
参考一下master分支的文档吧，0.12的文档没有修复。
https://iotdb.apache.org/zh/UserGuide/Master/Ecosystem%20Integration/Zeppelin-IoTDB.html

需要使用 `mvn clean package -pl zeppelin-interpreter -am -DskipTests -P get-jar-with-dependencies` 才会生成完整的jar包。
